content/docs/faq-structure.md

```diff
@@ -8,18 +8,67 @@ category: FAQ
 
 ### Is there a recommended way to structure React projects?
 
+React doesn't have opinions on how you put files into folders. That said there are a few common approaches popular in the ecosystem you may want to consider.
+
+#### Grouping by features or routes
+
 One common way to structure projects is locate CSS, JS, and tests together inside folders grouped by feature or route.
 
 ```
-FeatureA
+common/
+  Avatar.js
+  Avatar.css
+  APIUtils.js
+  APIUtils.test.js
+feed/
   index.js
-  ComponentA.js
-  ComponentA.css
-  ComponentA.test.js
-  Helper.js
-  Helper.test.js
-FeatureB
+  Feed.js
+  Feed.css
+  FeedStory.js
+  FeedStory.test.js
+  FeedAPI.js
+profile/
   index.js
-  ComponentB.js
-  ComponentB.test.js
+  Profile.js
+  ProfileHeader.js
+  ProfileHeader.css
+  ProfileAPI.js
+```
+
+The definition of what a "feature" is not universal, and it is up to you to choose the granularity. If you can't come up with a list of top-level folders, you can ask the users of your product what major parts it consists of, and use their mental model as a blueprint.
```

Comment:
There's an "is" missing here -> "The definition of what a feature **is** **is** not universal"

Reply:
> The definition of what a "feature" is not universal

The definition of what a "feature" **_can be_** is not universal ?